package.json

```diff
@@ -40,6 +40,7 @@
     "ip": "^1.1.5",
     "mailcheck": "^1.1.1",
     "passport": "^0.3.0",
+    "passport-apple": "https://github.com/relivecc/passport-apple#cbe68f547c1e49fd07999a105651de690f18feec",
```

Comment:
Let's make sure to leave a comment on that pull request to indicate we found the change helpful 👍